### ISSUE TITLE
Correct base text color to be $color-base and not $color-gray-dark

### ISSUE THREAD

**Issue by humancompanion-usds** [department-of-veterans-affairs/vets-design-system-documentation#935]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

I noticed when designing hint-text to be distinct from our form labels that our lables were $color-gray-dark and not $color-base as I had expected all base text to be (that's the whole point of $color-base). Instead I found that not only are our labels gray-dark but that's essentially our base content color. While $color-gray-dark passes AAA contrast, not having the base color be closer to black than it is to gray takes away some much needed hierarchy when we need to differentiate text from normal, base text (as we needed to do with hint-text). 

## What I expected to happen

I expected our base text color to be closer to black and have greater contrast with the white background. I also cannot tell from code if this decision was intentional or not. I consulted with Martha and she agrees that the darker the better.

## Reproducing
This can be reproduced on any device or browser.

Steps to reproduce:
1. Inspect va.gov on really any page with text.
2. Notice that we set the base color and then override it to $color-gray-dark.

![Screen Shot 2022-06-09 at 10 28 55 AM](https://user-images.githubusercontent.com/52176351/172871775-e99e221f-8a5c-4518-9832-22b71b1eafea.png)

**Comment by humancompanion-usds** [comment]
I'm confused as to why this is assigned to me. I gave clear direction yesterday to please make the change on color to $color-base. Is there some new information or a PR for review?

**Comment by caw310** [comment]
Jami thought she was waiting for additional feedback from you which is why I assigned to you. I think we're all set. @jamigibbs good on this now?

**Comment by jamigibbs** [comment]
@caw310 @humancompanion-usds Sorry I think I misunderstood where we left this yesterday. I will submit a PR for the Formation update.

**Comment by humancompanion-usds** [comment]
Comments in PR. Once @jamigibbs has decided with Brooks to remove .va-note and had the PR reviewed by Brooks then we can move forward and commit this change. Yay!
